docs/appendix/changelog.md

```diff
@@ -145,7 +145,7 @@ Use [`AnalogByName()`](/appendix/apis/components/board/#analogbyname) followed b
 
 `AnalogReader` has been renamed to `Analog`.
 The functionality remains the same, but code that uses analogs must be updated.
-`AnalogReaderByName()` and `AnalogReaderNames()` have become [`AnalogByName()`](/appendix/apis/components/board/#analogbyname) and [`AnalogNames()`](/appendix/apis/components/board/#analognames), respectively.
+`AnalogReaderByName()` and `AnalogReaderNames()` have become [`AnalogByName()`](/appendix/apis/components/board/#analogbyname) and `AnalogNames()`, respectively.
```

Comment:
You can't just remove the links here that makes this not make sense. We'll need to understand what happened and whether `AnalogNames()` actually existed. Or what the alternative we did replace it with is and add that instead.

Reply:
possibly link to https://python.viam.dev/autoapi/viam/components/board/client/index.html#viam.components.board.client.BoardClient.analog_names ?